packages/@glimmer/benchmark-env/src/benchmark/render-benchmark.ts

```diff
@@ -18,7 +17,7 @@ export default async function renderBenchmark(
   context: CompileTimeCompilationContext,
   runtimeResolver: RuntimeResolver,
   component: ResolvedComponentDefinition,
-  root: Dict,
```

Comment:
Is this just not needed?

Reply:
Correct, I refactored `renderComponent` to wrap the args internally, because that code was getting duplicated everywhere (I think 5 places now, including Glimmer.js). So now this wrapping happens internally, all you do is pass an object containing arguments to `renderComponent`.